pages/tutorials/yaml/phases.rst

```diff
@@ -0,0 +1,382 @@
+plus the composition. This state is specified as a mapping in the ``state``
+field of ``phase`` entry.
+
+The composition can be set using one of the following fields, depending on the
+phase type. The composition is specified as a mapping of species names to
+values. Where necessary, the values will be automatically normalized.
+
+- ``mass-fractions`` or ``Y``
+- ``mole-fractions`` or ``X``
+- ``coverages``
+- ``molalities`` or ``M``
+
+The thermodynamic state can be set in terms of two of the following properties,
+with the valid property pairs deplending on the phase model:
+
+- ``temperature`` or ``T``
+- ``pressure`` or ``P``
+- ``enthalpy`` or ``H``
+- ``entropy`` or ``S``
+- ``int-energy``, ``internal-energy`` or ``U``
+- ``specific-volume`` or ``V``
+- ``density`` or ``D``
+
+Examples:
+
+.. code:: yaml
+
+    state:
+      T: 300 K
+      P: 101325 Pa
+      X: {O2: 1.0, N2: 3.76}
+
+    state:
+      density: 100 kg/m^3
+      T: 298
+      Y:
+        CH4: 0.2
+        C3H8: 0.1
+        CO2: 0.7
+
+
+Examples
+--------
+
+An input file defining two equivalent gas phases including all reactions and
+species defined in the input file, (with species and reaction data elided):
+
+.. code:: yaml
+
+    phases:
+    - name: gas1
+      thermo: ideal-gas
+      elements: [O, H, N, Ar]
+      species: [H2, H, O, O2, OH, H2O, HO2, H2O2, N2, AR]
+      kinetics: gas
+      reactions: all
+      transport: mixture-averaged
+      state:
+        T: 300.0
+        P: 1.01325e+05
+    - name: gas2
+      thermo: ideal-gas
+      kinetics: gas
+      transport: mixture-averaged
+      state: {T: 300.0, 1 atm}
+
+    species:
+    - H2: ...
+    - H: ...
+    ...
+    - AR: ...
+
+    reactions:
+    ...
+
+An input file defining an interface and its adjacent bulk phases, with full
+species data elided:
+
+.. code:: yaml
+
+    phases:
+    - name: graphite
+      thermo: lattice
+      species:
+      - graphite-species: all
+      state: {T: 300, P: 101325, X: {C6: 1.0, LiC6: 1e-5}}
+      density: 2.26 g/cm^3
+
+    - name: electrolyte
+      thermo: lattice
+      species: [{graphite-anode-species: all}]
+      density: 1208.2 kg/m^3
+      state:
+        T: 300
+        P: 101325
+        X: {Li+(e): 0.08, PF6-(e): 0.08, EC(e): 0.28, EMC(e): 0.56}
+
+    - name: anode-surface
+      thermo: ideal-surface
+      kinetics: surface
+      reactions: [graphite-anode-reactions]
+      species: [{graphite-anode-species: [(int)]}]
+      site-density: 1.0 mol/cm^2
+      state: {T: 300, P: 101325}
+
+    graphite-species:
+    - name: C6
+      ...
+    - name: LiC6
+      ...
+
+    electrolyte-species:
+    - name: Li+(e)
+      ...
+    - name: PF6-(e)
+      ...
+    - name: EC(e)
+      ...
+    - name: EMC(e)
+      ...
+
+    anode-species:
+    - name: (int)
+      ...
+
+    graphite-anode-reactions:
+    - equation: LiC6 <=> Li+(e) + C6
+      rate-constant: [5.74, 0.0, 0.0]
+      beta: 0.4
```

Comment:
There's a lot of unexplored territory in electrochemistry in Cantera right now. I think as these capabilities are expanded, it will be easier to see what the best way of structuring the input is. So for the time being, I'm inclined to keep this as-is. There's nothing that would prevent us from requiring an explicit `type: electrochemical` field for electrochemical reactions in the future.